commons/src/test/java/org/sonarsource/analyzer/commons/RuleMetadataLoaderTest.java

```diff
@@ -94,7 +97,11 @@ class TestRule {
     assertThat(rule.severity()).isEqualTo("MINOR");
     assertThat(rule.type()).isEqualTo(RuleType.CODE_SMELL);
     assertThat(rule.cleanCodeAttribute()).isEqualTo(IDENTIFIABLE);
-    assertThat(rule.defaultImpacts()).isEqualTo(Map.of(MAINTAINABILITY, HIGH));
+    assertThat(rule.defaultImpacts()).isEqualTo(Map.ofEntries(
+      Map.entry(MAINTAINABILITY, HIGH),
+      Map.entry(SoftwareQuality.SECURITY, Severity.INFO),
```

Comment:
what about statically importing the constants (like for `MAINTAINABILITY` and `HIGH`) to improve readability and be consistent with other tests?